src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1049,6 +1051,39 @@ public void canCheckInLostAndPaidItem() {
     assertThat(itemsClient.getById(item.getId()).getJson(), isAvailable());
   }
 
+  @Test
+  public void itemCheckedInEventIsPublished() {
+    final IndividualResource james = usersFixture.james();
+    final UUID checkInServicePointId = servicePointsFixture.cd1().getId();
+    final IndividualResource homeLocation = locationsFixture.basedUponExampleLocation(
+      item -> item.withPrimaryServicePoint(checkInServicePointId));
+    final IndividualResource nod = itemsFixture.basedUponNod(item ->
+      item.withPermanentLocation(homeLocation.getId()));
+
+    DateTime checkOutDate = new DateTime(2020, 1, 18, 18, 0, 0, UTC);
+    DateTime checkInDate = new DateTime(2020, 1, 22, 15, 30, 0, UTC);
+
+    checkOutFixture.checkOutByBarcode(nod, james, checkOutDate);
+
+    CheckInByBarcodeResponse checkInResponse = checkInFixture.checkInByBarcode(
+      new CheckInByBarcodeRequestBuilder()
+        .forItem(nod)
+        .on(checkInDate)
+        .at(checkInServicePointId));
+
+    JsonObject checkedInLoan = checkInResponse.getLoan();
+
+    // There should be two events published - first one for "check out", second one for "check in"
+
+    List<JsonObject> publishedEvents = Awaitility.await()
```

Comment:
Personally, I would remove the blank line between the comment and the statement to reinforce that the comment is about that statement

Reply:
Removed blank lines in all similar places